pkg/cache/warm.go

```diff
@@ -59,35 +58,42 @@ func WarmCache(opts *config.WarmerOptions) error {
 
 	errs := 0
 	for _, img := range images {
-		tarBuf := new(bytes.Buffer)
-		manifestBuf := new(bytes.Buffer)
-
-		cw := &Warmer{
-			Remote:         remote.RetrieveRemoteImage,
-			Local:          LocalSource,
-			TarWriter:      tarBuf,
-			ManifestWriter: manifestBuf,
-		}
-
-		digest, err := cw.Warm(img, opts)
+		err := warmToFile(cacheDir, img, opts)
 		if err != nil {
-			if !IsAlreadyCached(err) {
-				logrus.Warnf("Error while trying to warm image: %v %v", img, err)
-				errs++
+			logrus.Warnf("Error while trying to warm image: %v %v", img, err)
+			errs++
+		}
+		/*
```

Comment:
I think this commented code block should be removed IIUC

Reply:
Ok, Thank you for the reply and the review. I removed the commented code block.